app/src/routers/auth.js

```diff
@@ -0,0 +1,60 @@
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import passport from 'passport';
+
+import authHelper from '../common/helpers/authHelper';
+import { secret } from '../config/jwt';
+
+const router = express.Router();
+
+router.post(
+    '/jwt', 
+    (req, res) => {
+        let resp = null;
+        const user = authHelper.isAuthorized(req.body.email, req.body.password);
+
+        if (user) {
+            const token = jwt.sign(JSON.stringify(user), secret.key);
+
+            resp = {
+                code: 200,
```

Comment:
Why do we need code and message in the response body?

Reply:
for the next homework))